src/components/BulkEditList/BulkEditList.js

```diff
@@ -23,16 +27,28 @@ export const BulkEditList = () => {
         defaultWidth="20%"
         paneTitle={<FormattedMessage id="ui-bulk-edit.list.criteriaTitle" />}
       >
-        <BulkEditListFilters />
+        <BulkEditListFilters
+          setFileUploadedName={setFileUploadedName}
+          setIsFileUploaded={setIsFileUploaded}
+          isFileUploaded={isFileUploaded}
+        />
       </Pane>
       <Pane
         defaultWidth="fill"
-        paneTitle={<FormattedMessage id="ui-bulk-edit.meta.title" />}
+        paneTitle={isFileUploaded ?
+          <FormattedMessage
+            id="ui-bulk-edit.meta.title.uploadedFile"
+            values={{ fileName: fileUploadedName }}
+          />
+          :
+          <FormattedMessage id="ui-bulk-edit.meta.title" />}
```

Comment:
Could you please move paneTitle expression in new variable? Render looks overloaded

Reply:
Thanks. I have fixed already